src/mxarray.jl

```diff
@@ -160,10 +165,11 @@ const _mx_is_struct  = mxfunc(:mxIsStruct)
 const _mx_is_cell    = mxfunc(:mxIsCell) 
 
 
-# getting simple attributes
-
 macro mxget_attr(fun, ret)
-    :( ccall($(fun)::Ptr{Void}, $(ret), (Ptr{Void},), mx.ptr) )
+    quote 
+        ptr = unsafe_convert(Ptr{Void}, mx)
```

Comment:
Don't call `unsafe_convert` directly. Please read the doc and the use case I linked.